### PR TITLE
update local font access API data

### PR DESCRIPTION
**Pull request by chrisdavidmills**
#### Summary

This PR updates the compact data for the [Local Font Access API](https://wicg.github.io/local-font-access/):

* The API features were already there, but the data was slightly wrong (the API is currently available only on Chromium desktop, not mobile).
* I also added data points for the `local-fonts` Permissions Policy header directive, and Permissions API permission.

See my research document for more details of exactly what the larger work involves: https://docs.google.com/document/d/1q1p_wlMe67LYvSuwbTZDOuL0csfG2rw_3qT2ROpC2sY/edit#

### CODE REVIEW

File: api/Permissions.json [hunk 1]
```diff
@@ -351,6 +351,49 @@
...
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
```
Comment (queengooborg) [inline api/Permissions.json:388]:
```suggestion
            "samsunginternet_android": "mirror",
            "webview_android": "mirror"
```

File: api/Window.json [hunk 2]
```diff
@@ -4060,15 +4062,23 @@
             "ie": {
               "version_added": false
             },
-            "oculus": "mirror",
-            "opera": "mirror",
+            "oculus": {
+              "version_added": false
+            },
```
Comment (queengooborg) [inline api/Window.json:4067]:
```suggestion
            "oculus": "mirror",
```

File: api/Window.json [hunk 3]
```diff
@@ -4060,15 +4062,23 @@
...
             },
             "safari_ios": "mirror",
-            "samsunginternet_android": "mirror",
-            "webview_android": "mirror"
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
```
Comment (queengooborg) [inline api/Window.json:4081]:
```suggestion
            "samsunginternet_android": "mirror",
            "webview_android": "mirror"
```

File: http/headers/Permissions-Policy.json [hunk 4]
```diff
@@ -731,6 +731,50 @@
...
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": {
+                "version_added": false
+              },
```
Comment (queengooborg) [inline http/headers/Permissions-Policy.json:755]:
```suggestion
              "oculus": "mirror",
```

File: http/headers/Permissions-Policy.json [hunk 5]
```diff
@@ -731,6 +731,50 @@
...
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
```
Comment (queengooborg) [inline http/headers/Permissions-Policy.json:769]:
```suggestion
              "samsunginternet_android": "mirror",
              "webview_android": "mirror"
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
There's a couple of places where we can keep mirroring (since Oculus, Samsung Internet and WebView mirror from Chrome Android), but otherwise LGTM!  The collector project agrees with these changes as well (see #18359 / https://github.com/mdn/browser-compat-data/blob/8a448979843f97afc3365ed7c4b441a512712ca8/api/FontData.json).  👍

**Comment by chrisdavidmills** [comment]
@queengooborg Good call, thanks for the suggestions. All committed!

**Review by queengooborg — APPROVED** [review]

**Pull request merged**